rcl_action/src/rcl_action/action_server.c

```diff
@@ -1240,6 +1240,40 @@ rcl_action_server_set_cancel_service_callback(
     user_data);
 }
 
+#define SERVER_CONFIGURE_SERVICE_INTROSPECTION(TYPE, STATE) \
+  if (rcl_service_configure_service_introspection( \
+      &action_server->impl->TYPE ## _service, \
+      node, \
+      clock, \
+      type_support->TYPE ## _service_type_support, \
+      publisher_options, \
+      STATE) != RCL_RET_OK) \
+  { \
+    return RCL_RET_ERROR; \
+  }
+
+rcl_ret_t
+rcl_action_server_configure_action_introspection(
+  rcl_action_server_t * action_server,
+  rcl_node_t * node,
+  rcl_clock_t * clock,
+  const rosidl_action_type_support_t * type_support,
+  const rcl_publisher_options_t publisher_options,
+  rcl_service_introspection_state_t introspection_state)
+{
```

Comment:
There is a warning in Windows CI:

`'rcl_action_server_configure_action_introspection': inconsistent dll linkage`

See reference build: 
* https://ci.ros2.org/job/nightly_win_rep/3657/

FYI: @Barry-Xu-2018 @fujitatomoya

Reply:
@Barry-Xu-2018 actually this was detected by https://ci.ros2.org/job/ci_windows/23428/msbuild/, but we overlooked it.

Reply:
@Crola1702 @Barry-Xu-2018 https://github.com/ros2/rcl/pull/1212 should fix this inconsistent linkage warning.